src/main/java/co/lotc/lever/cmd/Countdown.java

```diff
@@ -0,0 +1,37 @@
+package co.lotc.lever.cmd;
+
+import static org.bukkit.ChatColor.*;
+
+import co.lotc.core.bukkit.util.Run;
+import co.lotc.core.command.annotate.Arg;
+import co.lotc.core.command.annotate.Default;
+import co.lotc.core.command.annotate.Range;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+
+import co.lotc.core.bukkit.util.LocationUtil;
+import co.lotc.lever.BaseCommand;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Countdown extends BaseCommand {
+    private static final int BROADCAST_RADIUS = 20;
+
+    public void invoke(Player source, @Arg("Timer") @Default("5") @Range(min=5,max=60) int timer) {
+        List<Player> PlayersNearby = new ArrayList<>();
+        Bukkit.getOnlinePlayers().stream()
+                .filter(player->LocationUtil.isClose(source, player, BROADCAST_RADIUS))
+                .forEach(PlayersNearby::add);
```

Comment:
I'm not 100% about this approach. This means players who move into range after the countdown started won't be receiving messages

Reply:
same, but i figured people would already know if someone was going to start a countdown :p